digiwf-apps/packages/apps/digiwf-tasklist/src/views/Processes.vue

```diff
@@ -132,6 +162,68 @@ export default class Processes extends Vue {
     return this.processDefinitions.filter(task => JSON.stringify(Object.values(task)).toLocaleLowerCase().includes(this.filter.toLocaleLowerCase()));
   }
 
+  get isFilterPersistent(): boolean {
+    if (
+      !this.filter ||
+      this.filter.length == 0 ||
+      !this.persistentFilters ||
+      this.persistentFilters!.length == 0
+    ) {
+      return false;
+    }
+    return (
+      this.persistentFilters!.find(
+        (fl: FilterTO) => fl.filterString == this.filter
+      ) != undefined
+    );
+  }
+
+  async savePersistentFilter() {
+    if (!this.filter){
+      return;
+    }
+    const request: SaveFilterTO = {
+      pageId: "processes",
+      filterString: this.filter,
+    }
+    try {
+      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getPUTConfig({}));
+      await FilterRestControllerApiFactory(cfg).saveFilter(request);
+
+      this.errorMessage = "";
+      this.$store.dispatch('filters/getFilters', true);
+    } catch (error) {
+      this.errorMessage = 'Der Filter konnte nicht gespeichert werden.';
+    }
+  }
+
+  async deletePersistentFilter() {
+    const id = this.persistentFilters!.find((f: FilterTO) => f.filterString == this.filter)?.id!
+    try {
+      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getDELETEConfig());
+      await FilterRestControllerApiFactory(cfg)._delete(id);
+
+      this.errorMessage = "";
+      this.$store.dispatch('filters/getFilters', true);
+    } catch (error) {
+      this.errorMessage = 'Der Filter konnte nicht gelöscht werden.';
+    }
+  }
+
+  async loadPersistentFilters(refresh = false): Promise<void> {
+    this.persistentFilters = this.$store.getters['filters/filters'].filter((filter: FilterTO) => filter.pageId === "processes");
+    try {
+      await this.$store.dispatch('filters/getFilters', refresh);
+      this.errorMessage = "";
+    } catch (error) {
+      this.errorMessage = error.message;
+    }
+  }
```

Comment:
The code block with the api calls is exactly the same as in tasklist.vue and processinstance.vue. Should we move this into a seperate component or function?